drivers/platform/aducm3029/irq_extra.h

```diff
@@ -110,6 +114,28 @@ struct rtc_irq_config {
 	uint32_t active_interrupts;
 };
 
+/**
+ * @enum gpio_irq_mode
+ * @brief Interrupt conditions for GPIO group interrupts
+ */
+enum gpio_irq_mode {
+	/** Falling edge */
+	GPIO_GROUP_NEGATIVE_EDGE,
+	/** Rising edge */
+	GPIO_GROUP_POSITIVE_EDGE
+};
+
+/**
+ * @struct gpio_irq_config
+ * @brief GPIO group interrupt configuration routine.
+ */
+struct gpio_irq_config {
+	/** GPIO driver handler */
+	struct gpio_desc *gpio_handler;
```

Comment:
nitpick: you could indent the structure fields for clarity

Reply:
Will do.